src/ops.rs

```diff
@@ -177,6 +203,13 @@ impl<A: BitSetLike> BitSetLike for BitSetNot<A> {
     }
 }
 
+impl<A: DrainableBitSet> DrainableBitSet for BitSetNot<A> {
+    #[inline]
+    fn remove(&mut self, _: Index) -> bool {
+        false
```

Comment:
Does this make sense? By itself it's pretty useless, but if you combine with other ones it could be useful.

Reply:
Wouldn't it be more straightforward to actually add a bit in `BitSetNot::remove` instead of returning a bool?

Reply:
`(!b).drain()` would fill the bit set? I guess that's a possibility.

Reply:
Well it would require adding `add` method to the trait. And the trait should be renamed. (I think it should be right now too as `remove` doesn't really have anything inherently common with draining iterator)

Reply:
Ah now I remember why I wanted to avoid adding `add` method: It doesn't make any sense for `xor` and it's bit iffy for `or`.

Reply:
How about simply *not* implementing `DrainableBitSet` for not/xor/or since it doesn't make sense?

Reply:
I think drain/removal makes sense for both  `or` and `and`. I'll remove it from others.